iml-services/iml-device/src/main.rs

```diff
@@ -87,10 +93,32 @@ async fn main() -> Result<(), ImlDeviceError> {
 
     let mut s = consume_data("rust_agent_device_rx");
 
-    while let Some((fqdn, device)) = s.try_next().await? {
+    let pool = iml_orm::pool().unwrap();
+
+    while let Some((f, d)) = s.try_next().await? {
         let mut cache = cache2.lock().await;
 
-        cache.insert(fqdn, device);
+        // Specify the type, otherwise inference fails on `.clone()`
+        let d: Device = d;
```

Comment:
```suggestion
```

I removed these lines and compilation seems ok, Could be because I removed the clone below.